### PR TITLE
Used new version_compare() function to simplify version related logic

### PR DESCRIPTION
**Pull request by olehermanse**
@minggo Please test/review. I cannot test windows code.

### CODE REVIEW

File: bin/utils.py [hunk 1]
```diff
@@ -165,55 +165,45 @@ def get_vs_versions():
...
+        v_path = get_devenv_path(cur_ver)
         if v_path is not None:
-            if (find_ver is None) or (cur_v > find_ver):
-                find_ver = cur_v
+            if (find_ver is None) or cmp(cur_ver, ">", find_ver):
+                find_ver = cur_ver
                 find_path = v_path
 
-    if min_ver_float > 0 and find_ver > min_ver_float:
+    if cmp(min_ver_float, ">", 0) and cmp(find_ver, ">", min_ver_float):
```
Comment (minggo) [inline bin/utils.py]: min_ver_float -> min_ver?

Comment (olehermanse) [reply bin/utils.py]: Thanks, fixed.

**Comment by olehermanse** [comment]
@minggo Do you wanna merge this, or should I close?

**Comment by minggo** [comment]
Oh sorry, i will merge it after review it. Please send me a message after you modify something in future, thanks.

**Comment by minggo** [comment]
It is merged, could you please update cocos2d-x's submodule? Thanks.

**Pull request merged**